### PR TITLE
Set up project

### LINKED ISSUE #1: First time setup project
**Is your feature request related to a problem? Please describe.**
Yes, setting up the project is needed to get started.

**Describe the solution you'd like**
Using 5 subdirectories for the different services:
- Customer Service (@JustMilan)
- Product Service (@Rubenvdbrink)
- Order Service (@JortWillemsen)
- Review Service (@xandervedder)
- Delivery Service (@arjennorbart)

Use Docker in the root the start all the different projects.

**Describe alternatives you've considered**
\-

**Additional context**
https://canvas.hu.nl/courses/27226/assignments/171154

### PR DESCRIPTION
**Pull request by xandervedder**
## Changes
Initial project setup. Not sure if it was done correctly, hence why it is still a draft.

## Checklist
- [x] Create initial project setup
- [x] Check if this the right way to add modules to the project
- [x] Configure docker to start RabbitMQ
- [x] Configure docker to start MongoDB
- [x] ~~Configure docker to start Spring applications~~
  not going to do this one, it can be done in intellij instead (configuring hot reload in intellij is also easier.
- [x] change location of docker-compose (maybe in root instead of development).

## When merging, this pull request 
closes #1

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -0,0 +1,2 @@
+.idea/
+development/**/volume/
```
Comment (JustMilan) [inline .gitignore:2]:
```suggestion
development/**/volume/

```

No newline eof

File: Customer/pom.xml [hunk 2]
```diff
@@ -0,0 +1,54 @@
...
+    <modelVersion>4.0.0</modelVersion>
+    <parent>
+        <groupId>org.springframework.boot</groupId>
+        <artifactId>spring-boot-starter-parent</artifactId>
+        <version>2.6.0</version>
+        <relativePath/> <!-- lookup parent from repository -->
+    </parent>
+    <groupId>nl.softwarestrijders.waiter</groupId>
+    <artifactId>Customer</artifactId>
+    <version>0.0.1-SNAPSHOT</version>
```
Comment (JustMilan) [inline Customer/pom.xml:13]: Is this correct?

File: Customer/src/test/java/nl/softwarestrijders/waiter/customer/CustomerApplicationTests.java [hunk 3]
```diff
@@ -0,0 +1,13 @@
...
+
+import org.junit.jupiter.api.Test;
+import org.springframework.boot.test.context.SpringBootTest;
+
+@SpringBootTest
+class CustomerApplicationTests {
+
+    @Test
+    void contextLoads() {
+    }
```
Comment (JustMilan) [inline Customer/src/test/java/nl/softwarestrijders/waiter/customer/CustomerApplicationTests.java:11]: Isn't this test unneccesary?

File: .gitignore (hunk 1 above)
Comment (xandervedder) [reply .gitignore:2]: Yes. I was debating doing something like this instead:

```suggestion
development/*
```
And moving the docker-compose.yml in the root of the project

File: Customer/src/test/java/nl/softwarestrijders/waiter/customer/CustomerApplicationTests.java (hunk 3 above)
Comment (xandervedder) [reply Customer/src/test/java/nl/softwarestrijders/waiter/customer/CustomerApplicationTests.java:11]: This was all created by Spring Intializr. I'm not sure if tests are mandatory or not (for grading).

File: Customer/pom.xml (hunk 2 above)
Comment (xandervedder) [reply Customer/pom.xml:13]: Yes, the project is called Waiter internally. That's why there's a `waiter` package for all of them (it looks like `src/main/java/nl/softwarestrijders/waiter/customer/CustomerApplication.java`)

**Comment by Rubenvdbrink** [comment]
LGTM

**Review by Rubenvdbrink — APPROVED** [review]

**Review by JortWillemsen — APPROVED** [review]

**Pull request merged**